src/frontend/Ast_to_Mir.ml

```diff
@@ -770,11 +763,15 @@ let%expect_test "read data" =
      (For (loopvar sym1__) (lower (Lit Int 1)) (upper (Lit Int 5))
       (body
        (Block
-        ((Assignment (mat ((Single (Var sym1__))))
-          (Indexed
-           (FunApp CompilerInternal FnReadData__
-            ((Lit Str mat) (Lit Str matrix) (Lit Int 10) (Lit Int 20)))
-           ((Single (Var sym1__)))))))))) |}]
+        ((For (loopvar sym2__) (lower (Lit Int 1))
```

Comment:
I see that indexing for loops in the MIR also starts at 1 (rather than 0, as C++ likes it). Is that intentional?

Reply:
Yeah, right now I'm keeping the MIR indexed to 1 everywhere (or trying to). This seemed a little easier because we often want to print out the index in terms the user expects.